pkg/config/deployment_kubernetes.go

```diff
@@ -38,6 +38,9 @@ type KubernetesDeploymentSpec struct {
 
 // Validate returns an error if any wrong configuration value was found.
 func (s *KubernetesDeploymentSpec) Validate() error {
+	if err := s.GenericDeploymentSpec.Validate(); err != nil {
+		return err
+	}
```

Comment:
```go
return s.GenericDeploymentSpec.Validate()
```

Reply:
Let me keep this as is. Because other fields of `KubernetesDeploymentSpec` should be validated too. (at other PRs)

Reply:
Makes sense.

Reply:
I see, thanks 👍